### ISSUE TITLE
The tooltip select merges with the text

### ISSUE THREAD

**Issue by max-programming** [codeSTACKr/codestackr-vscode-theme#3]
Just noticed that the when we select the text in the tooltip, it merges. A lighter colour for this would be nice
![image](https://user-images.githubusercontent.com/51731966/93222479-a1304a00-f78c-11ea-9986-93c40692e6f8.png)

**Comment by codeSTACKr** [comment]
Sorry for the late reply. Is this still an issue?

**Comment by max-programming** [comment]
Yes. The selection color makes the text hard to read. Thanks for taking this into consideration
